utilities/genSeqErrorModel.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
-
 #
 #
-#          genSeqErrorModel.py
-#          Computes sequencing error model for gen_reads.py
+#   genSeqErrorModel.py
+#   Computes sequencing error model for gen_reads.py
+#
+#   What is input -> output       
```

Comment:
@joshfactorial can you add it here what type of inputs it can take?

It first seemed like it takes a Bam/Sam file but seems like it can take Fasta/Fastq as well, little confused